vespa/application.py

```diff
@@ -979,9 +1005,10 @@ async def update_data(
         fields: Dict,
         create: bool = False,
         namespace: str = None,
+        groupname:str=None,
         **kwargs
     ) -> VespaResponse:
-        path = self.app.get_document_v1_path(id=data_id, schema=schema, namespace=namespace)
+        path = self.app.get_document_v1_path(id=data_id, schema=schema, namespace=namespace, group=groupname)
```

Comment:
Just string groups? I guess that's the most common use-case.

Reply:
According to doc, group is the way https://docs.vespa.ai/en/streaming-search.html#using-streaming-search

Reply:
👍